core/src/main/java/com/microsoft/applicationinsights/internal/channel/samplingV2/FixedRateSamplingTelemetryProcessor.java

```diff
@@ -0,0 +1,212 @@
+package com.microsoft.applicationinsights.internal.channel.samplingV2;
+
+import com.microsoft.applicationinsights.agent.internal.common.StringUtils;
+import com.microsoft.applicationinsights.extensibility.TelemetryProcessor;
+import com.microsoft.applicationinsights.internal.annotation.BuiltInProcessor;
+import com.microsoft.applicationinsights.internal.logger.InternalLogger;
+import com.microsoft.applicationinsights.telemetry.SupportSampling;
+import com.microsoft.applicationinsights.telemetry.Telemetry;
+import org.omg.CORBA.INTERNAL;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * This processor is used to Perform Sampling on User specified sampling rate
+ * <p>
+ * How to use in ApplicationInsights Configuration :
+ * <p>
+* <TelemetryProcessors>
+    <BuiltInProcessors>
+        <Processor type = "FixedRateSamplingTelemetryProcessor">
+            <Add name = "SamplingPercentage" value = "50" />
+            <ExcludedTypes>
+                <excludedType>Request</excludedType>
```

Comment:
Making generic `Add` instead of `excludedType` will allow to generalize this logic in future

Reply:
There is already a separate add tag in the source tag. While I agree with your point that this will add more extensibility but I thought that being specific here adds a lot more ease of understanding and separation of concern. We can discuss on deciding this.